docs/cimplInstall.md

```diff
@@ -234,16 +250,18 @@ With [homebrew](#homebrew) installed, yarn can be installed by entering the foll
 ```
 brew install yarn
 ```
+**Close and re-open your command line tool** to make sure those settings take effect
```

Comment:
Remove this. closing and reopening a terminal session is not necessary in Mac OS.